common/key.c

```diff
@@ -247,10 +302,10 @@ v_key_name(SCR *sp, ARG_CHAR_T ach)
 			goto nopr;
 
 	/*
-	 * Historical (ARPA standard) mappings.  Printable characters are left
-	 * alone.  Control characters less than 0x20 are represented as '^'
-	 * followed by the character offset from the '@' character in the ASCII
-	 * character set.  Del (0x7f) is represented as '^' followed by '?'.
+	 * Printable characters are left alone. Control characters
+	 * less than 0x20 are represented as human readable <C-char>
+	 * notation. Carriage feed, escape and delete marked as
+	 * <Enter>, <Esc> and <Del> respectively.
```

Comment:
There needs to be a new option to turn this mode on. Checkout how the options such as `octal` and `noprint` work.

Reply:
Fixed.

Reply:
I made some changes including renaming the option. Can you give it a review?

Reply:
I looked at it, everything ok. I like builtin unreachable, but it sad, that this is doesn't break program execution. Anyway, it unlikely will bring any problems. Thank your for changes.